config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+
+  root               'static_pages#home'
+  get   'help'    => 'static_pages#help'
+  get   'about'   => 'static_pages#about'
+  get   'contact' => 'static_pages#contact'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.